src/schema/validator.js

```diff
@@ -205,17 +203,23 @@ export class SchemaValidator {
     this.allowedManifestVersionsRange =
       getManifestVersionsRange(validatorOptions);
 
-    const validator = ajv({
+    const validator = new Ajv({
+      strict: false,
       allErrors: true,
-      errorDataPath: 'property',
-      jsonPointers: true,
+      // include schema and data properties in error objects.
       verbose: true,
-      schemas: this.schemas,
-      schemaId: 'auto',
+      // TODO: do the following options have new replacements in ajv v8?
+      // what was the reason for setting these options in older ajv version?
+      //
+      // errorDataPath: 'property',
```

Comment:
[This commit](https://github.com/ajv-validator/ajv/commit/ef40fbbda6f165d95a82dddb9103d6475b0fd130) says:

>  - _errorDataPath_ (deprecated): set `dataPath` to point to 'object' (default) or to 'property' when validating keywords `required`, `additionalProperties` and `dependencies`.

But that has been deprecated a while ago and removed in v8 so I am not sure it is super useful these days. If that does not break any test, we should be good?

Reply:
The instancePath is included and so we should be good on that side, but now that I re-read that point I think that it may be actually the change that is triggering the different behavior with what the instancePath to the parent object (what I've been describing in https://github.com/mozilla/addons-linter/pull/4263#discussion_r860237509).

Reply:
I just re-tried what happens if I try to pass it, it throws and so I have removed the comment, it is likely related to the change in behavior that I did mention in the previous comment, but I guess that we have to accept this change in behavior.